pkg/collect/run_pod.go

```diff
@@ -58,6 +60,33 @@ func (c *CollectRunPod) Collect(progressChan chan<- interface{}) (CollectorResul
 		if err := client.CoreV1().Pods(pod.Namespace).Delete(context.Background(), pod.Name, metav1.DeleteOptions{}); err != nil {
```

Comment:
Nit pick: This function is big enough to be extracted into its own method in `CollectRunPod` class. `func (c *CollectRunPod) DeletePod()` perhaps?